prow/cmd/deck/static/prow/prow.ts

```diff
@@ -645,23 +645,17 @@ function redraw(fz: FuzzySearch): void {
         max = 2 * 3600;
     }
     drawJobHistogram(totalJob, jobHistogram, now - (12 * 3600), now, max);
-    if (rerunStatus === "error") {
+    if (rerunStatus === "gh_redirect") {
         modal.style.display = "block";
-        rerunCommand.innerHTML = "You don't have permission to rerun that job";
-    } else if (rerunStatus === "success") {
-        modal.style.display = "block";
-        rerunCommand.innerHTML = "Job successfully triggered. Wait 30 seconds and refresh the page for the job to show up";
-    } else if (rerunStatus != null) {
-        modal.style.display = "block";
-        rerunCommand.innerHTML = "Nice try! The direct rerun feature hasn't been implemented yet, so that button does nothing.";
+        rerunCommand.innerHTML = "Rerunning that job requires GitHub login. Now that you're logged in, try again";
     }
 }
 
 function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string): HTMLTableDataCellElement {
     const url = `${location.protocol}//${location.host}/rerun?prowjob=${prowjob}`;
     const c = document.createElement("td");
     const i = icon.create("refresh", "Show instructions for rerunning this job");
-    const login = getCookieByName("access-token-session");
+    const login = getCookieByName("github_login");
```

Comment:
We actually want to know whether the `access-token-session` cookie exists, but it is secure and http only, so it doesn't always show up in `document.cookie`. `github_login` is always set when `access-token-session` is set, and they have the same max age, so by checking for `github_login`  we can determine whether `access-token-session` has been set.

Reply:
This seems like good info for a comment?